source/index.txt

```diff
@@ -1,18 +1,18 @@
-.. _mongoid:
+.. _mongoid-odm:
 
-*******
-Mongoid
-*******
+=======
+{+odm+}
+=======
 
-.. default-domain:: mongodb
-
-Mongoid is the officially supported object-document mapper (ODM) for MongoDB in
-Ruby. To work with Mongoid from the command line using ``rails``-like tooling,
-the `railsmdb <https://github.com/mongodb/mongoid-railsmdb>`_ utility can be used.
+{+odm+} is the officially supported object-document mapper (ODM) for
+MongoDB in Ruby. To work with {+odm+} from the command line using
+``rails``-like tooling, the `railsmdb
+<https://github.com/mongodb/mongoid-railsmdb>`_ utility can be used.
```

Comment:
S: change to active voice

```suggestion
``rails``-like tooling, you can use the `railsmdb
<https://github.com/mongodb/mongoid-railsmdb>`_.
```

Reply:
Ah, just left this as placeholder content until the index ticket